locale/ru/openwith.dtd

```diff
@@ -56,10 +56,10 @@
 
 <!-- These strings appear in the keyboard shortcut dialog. In about:openwith,
      choose a browser, then Edit, Keyboard Shortcut. -->
-<!ENTITY openwith-ctrlkey "Control">
-<!ENTITY openwith-cmdkey "Command">
-<!ENTITY openwith-shiftkey "Shift">
-<!ENTITY openwith-altkey "Alt">
-<!ENTITY openwith-nomodifier "Select at least one modifier.">
-<!ENTITY openwith-comboinuse "This key combination appears to be in use.">
-<!ENTITY openwith-removecombo "Remove Shortcut">
+<!ENTITY openwith-ctrlkey "Управление">
+<!ENTITY openwith-cmdkey "Комманда">
+<!ENTITY openwith-shiftkey "Кнопка Shift">
+<!ENTITY openwith-altkey "Кнопка Alt">
```

Comment:
Alt

// if there's nothing with "Button " in front of the button name in the English localization then those shouldn't be added in Russian.